File: Pocs/Automated Test/AgileContent/NetCoreStandard/NetCoreSeleniumTest/Pages/LoginPage.cs
using NetCoreSeleniumTest.Model;
using OpenQA.Selenium;

namespace NetCoreSeleniumTest.Pages
{
    public class LoginPage: BasePage
    {
        private By frameLocator;
        private By numeroDelineaLocator;
        private By clavePersonalLocator;
        private By ingresarLocator;

        public LoginPage(IWebDriver driver) :base(driver) 
        {
            frameLocator = By.Id("loginIframe");
            numeroDelineaLocator = By.Id("idToken1");
            clavePersonalLocator = By.Id("idToken2");
            ingresarLocator = By.Id("loginButton_0");
        }

        public void Login(User user)
        {
            var frame = GetElement(frameLocator);
            _driver.SwitchTo().Frame(frame);

            GetElement(numeroDelineaLocator).SendKeys(user.Username);

            GetElement(clavePersonalLocator).SendKeys(user.Password);

            GetElement(ingresarLocator).Click();

            _driver.SwitchTo().DefaultContent();
        }
    }
}

File: Pocs/Automated Test/AgileContent/Node.js/SeleniumMocha/Personal/Pages/login-page.js
const {By} = require('selenium-webdriver');
const BasePage = require('./base-page');

class LoginPage extends BasePage {

  constructor(driver) {
    super(driver);
    this.frameLocator = By.id('loginIframe');
    this.numeroDeLineaLocator = By.id('idToken1');
    this.clavePersonalLocator = By.id('idToken2');
    this.ingresarLocator = By.id('loginButton_0');
  }

  async login(credentials) {
    const frame = await this.getElement(this.frameLocator);
    await this.driver.sleep(2000); // ¯\_(ツ)_/¯ 
    await this.driver.switchTo().frame(frame);
    await this.driver.sleep(2000); // ¯\_(ツ)_/¯

    await this.getElement(this.numeroDeLineaLocator).sendKeys(credentials.username);
    await this.getElement(this.clavePersonalLocator).sendKeys(credentials.password);
    await this.getElement(this.ingresarLocator).click();

    await this.driver.switchTo().defaultContent();
  }
}

module.exports = LoginPage;